discord/activity.py

```diff
@@ -138,7 +138,7 @@ def __init__(self, **kwargs):
         self.timestamps = kwargs.pop('timestamps', {})
         self.assets = kwargs.pop('assets', {})
         self.party = kwargs.pop('party', {})
-        self.application_id = kwargs.pop('application_id', None)
+        self.application_id = int(kwargs.pop('application_id', None))
```

Comment:
If there is no `application_id` and it returns `None` then you do `int(None)` which will raise an exception.

Reply:
This is why there is a helper for this code -- `utils._get_as_snowflake`

Reply:
I did not know about this function until now and have now updated my code.